conf/application.conf

```diff
@@ -47,7 +47,7 @@ cas {
 
 get-address-io-api {
     url = "https://api.getAddress.io/v2/uk/"
-    key = "12Co4w5VdUGJFpGcf28nhQ5877"
+    key = "q0M6MVJQm0CslqDvMcKUDQ6396"
```

Comment:
Yeah the old dev one expired so I just changed it to a new one I created. It'll be valid for 2 weeks.